packages/amplify-cli/src/utils/post-install-initialization.ts

```diff
@@ -0,0 +1,55 @@
+import { GetPackageAssetPaths, pathManager } from 'amplify-cli-core';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import { print } from '../context-extensions';
+
+/**
+ * This function is run first thing after a new Amplify installation
+ * It copies some assets needed by dependency packages to the .amplify folder
+ */
+export const postInstallInitialization = async () => {
+  print.info(`Initializing new Amplify CLI version...`);
+  // clean any previous libs
+  await fs.remove(pathManager.getAmplifyLibRoot());
+  await fs.ensureDir(pathManager.getAmplifyLibRoot());
+
+  // set env var to prevent errors due to no package libs
+  process.env.AMPLIFY_SUPPRESS_NO_PKG_LIB = 'true';
+
+  await Promise.all(
+    copyPkgAssetRegistry.map(async packageName => {
+      const { getPackageAssetPaths } = (await import(packageName)) as { getPackageAssetPaths: GetPackageAssetPaths };
+      if (typeof getPackageAssetPaths !== 'function') {
+        return;
+      }
+      const pluginArtifactPaths = await getPackageAssetPaths();
+      if (!Array.isArray(pluginArtifactPaths)) {
+        return;
+      }
+      await Promise.all(
+        pluginArtifactPaths.map(assetPath =>
+          fs.copy(
+            path.join(resolvePackageRoot(packageName), assetPath),
+            path.join(pathManager.getAmplifyPackageLibDirPath(packageName), assetPath),
+          ),
+        ),
+      );
+    }),
+  );
+  delete process.env.AMPLIFY_SUPPRESS_NO_PKG_LIB;
+  print.success('Done');
```

Comment:
"Done copying over plugin dependencies"?

Reply:
changed to "Done initializing new version."